src/mixins/map-mixin.js

```diff
@@ -196,6 +198,17 @@ export default function mapMixin(
     return _chart
   }
 
+  _chart.polyDim = function(polyDim) {
```

Comment:
Can polyDim be a function here? Will it cause any problems if it is?

Reply:
@jrajav polyDim is a function.